target/linux/rockchip/Makefile

```diff
@@ -0,0 +1,26 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+ARCH:=arm
+BOARD:=rockchip
+BOARDNAME:=Rockchip RK33xx
+FEATURES:=ext4 audio usb usbgadget display gpio fpu rootfs-part boot-part
+SUBTARGETS:=rk3399
+
+KERNEL_PATCHVER=5.4
+
+define Target/Description
+	Build firmware image for Rockchip RK3399 SoC devices.
+endef
+
+include $(INCLUDE_DIR)/target.mk
+
+DEFAULT_PACKAGES += \
+	uboot-envtools partx-utils e2fsprogs
```

Comment:
If it's limited to those few, you may put everything in one line ...

Reply:
Yup, done.